src/pages/valvonta-kaytto/valvonta-api.js

```diff
@@ -39,6 +39,16 @@ export const url = {
       id,
       toimenpideId
     )}/yritykset/${yritysId}/document/${filename}`,
+  courtAttachmentHenkilo: (henkiloId, id, toimenpideId) =>
+    `${url.toimenpide(
+      id,
+      toimenpideId
+    )}/henkilot/${henkiloId}/attachment/hallinto-oikeus.pdf`,
+  courtAttachmentYritys: (yritysId, id, toimenpideId) =>
+    `${url.toimenpide(
+      id,
+      toimenpideId
+    )}/yritykset/${yritysId}/attachment/hallinto-oikeus.pdf`,
```

Comment:
Tämä on tuota edellistä mietintää:

Tässä olisi siis `courtAttachmentYritys` sijaan `attachmentYritys`, joka ottaisi myös `attachmentName`

Reply:
Olisi kyllä varmaan vähän premature optimization, eikä ehkä myöskään "Keep it simple" 🤔

Reply:
Mutta siis ehdottomasti näin sitten, jos meille tulee muita liitteitä.